src/index.html

```diff
@@ -6,9 +6,31 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo">
+        <a href="index.html">
+          <img src="images/logo.png" alt="moyo logo">
+        </a>
+      </div>
+      <nav class="nav_menu">
+        <ul>
+          <li class="link"><a class="is-active" href="#">Apple</a></li>
```

Comment:
apply these foxes for other items below as well
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item"><a class="nav__link is-active" href="#">Apple</a></li>
```